src/gluonts/dataset/split.py

```diff
@@ -152,10 +152,16 @@ def slice_data_entry(
     if slice_.start is not None:
         offset = slice_.start
         if offset < 0:
-            offset += entry["target"].shape[0]
+            offset += entry["target"].shape[-1]
         sliced_entry[FieldName.START] += offset
 
-    sliced_entry[FieldName.TARGET] = sliced_entry[FieldName.TARGET][slice_]
+    # TODO fix
```

Comment:
What would the fix be?

Reply:
I was thinking of getting rid of the branching in favor of constructing a better `slice_` that would work in all cases. But it's OK for now